test/cluster/test_view_building_coordinator.py

```diff
@@ -189,20 +189,20 @@ async def test_add_view_while_build_in_progress(manager: ManagerClient):
     async with new_test_keyspace(manager, f"WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 3}} AND tablets = {{'enabled': true}}") as ks:
         await cql.run_async(f"CREATE TABLE {ks}.tab (key int, c int, v text, PRIMARY KEY (key, c))")
         await populate_base_table(cql, ks, "tab")
- 
+
         marks = await mark_all_servers(manager)
         await pause_view_building_tasks(manager)
- 
+
         await cql.run_async(f"CREATE MATERIALIZED VIEW {ks}.mv_cf_view1 AS SELECT * FROM {ks}.tab "
                         "WHERE c IS NOT NULL and key IS NOT NULL AND v IS NOT NULL PRIMARY KEY (c, key, v) ")
-         
+
         await wait_for_some_view_build_tasks_to_get_stuck(manager, marks)
- 
+
         await cql.run_async(f"CREATE MATERIALIZED VIEW {ks}.mv_cf_view2 AS SELECT * FROM {ks}.tab "
                         "WHERE c IS NOT NULL and key IS NOT NULL AND v IS NOT NULL PRIMARY KEY (c, key, v) ")
- 
+
         await unpause_view_building_tasks(manager)
- 
+
```

Comment:
There are lots of whitespace issues in the last commit. Please clean the commit up:
- If the whitespace issues were introduced in previous commits, please resolve them there
- If those are preexisting whitespace issues, I suggest leaving it for now. Alternatively, you can send a separate PR which cleans those issues up (especially if those issues were caused e.g. by an automatic whitespace cleaning up tool in your editor)